### ISSUE TITLE
Chainable transforms

### ISSUE THREAD

**Issue by dallasread** [Mike96Angelo/Bars#12]
Something like `{{@last(events)/id}}` would be neat.

**Comment by dallasread** [comment]
In the mean time, I've been using this transform:

```
function last(arr, path) {
    var item = arr[arr.length - 1],
        splat = path.split('.'),
        lastKey = splat.pop();

    for (var i = 0; i < splat.length; i++) {
        if (typeof item[splat[i]] !== 'object') return;
        item = item[splat[i]];
    }

    return item[lastKey];
}
```

with `{{@last(events, 'id')}}` OR EVEN `{{@last(events, 'some.crazy.path.id')}}`

**Comment by Mike96Angelo** [comment]
It in implemented v0.9.0+
